src/igvfd/tests/test_audit_file_set.py

```diff
@@ -151,3 +151,49 @@ def test_audit_single_cell_read_names(testapp, measurement_set_one_onlist, seque
         error['category'] == 'unexpected read names'
         for error in res.json['audit'].get('NOT_COMPLIANT', [])
     )
+
+
+def test_audit_control_for_control_type(
+    testapp,
+    measurement_set_mpra,
+    construct_library_set_reporter,
+    analysis_set_base
+):
+    res = testapp.get(measurement_set_mpra['@id'] + '@@audit')
+    assert all(
+        error['category'] != 'missing control for'
+        for error in res.json['audit'].get('NOT_COMPLIANT', [])
+    )
```

Comment:
I think it would make sense to add another assertion here that there is no "missing control type" audit present either.

Reply:
done